packages/forms/src/elements/common/Field.tsx

```diff
@@ -5,18 +5,23 @@
  * found at http://www.apache.org/licenses/LICENSE-2.0.
  */
 
-import React from 'react';
+import React, { HTMLAttributes } from 'react';
 import PropTypes from 'prop-types';
 import { useField } from '@zendeskgarden/container-field';
 import { FieldContext } from '../../utils/useFieldContext';
 import { StyledField } from '../../styled';
 
+export interface IFieldProps extends HTMLAttributes<HTMLDivElement> {
+  /** Field ID and prefix for generated label, input, and hint IDs */
+  id?: string;
+}
+
 /**
  * Provides layout and ID context for child components; accepts all `<div>`
  * attributes and events.
  */
-const Field = React.forwardRef((props, ref) => {
-  const getMessageProps = messageProps => ({ role: 'alert', ...messageProps });
+export const Field = React.forwardRef<HTMLDivElement, IFieldProps>((props, ref) => {
+  const getMessageProps = (messageProps: any) => ({ role: 'alert', ...messageProps });
```

Comment:
Should `messageProps` be typed?

Reply:
Since this is defined inline it's much more difficult to type it with generics. With this only being used internally I didn't want to rewrite the logic to make it fit our prop-getter structure more easily. If it's something you think we need I can dig into it some more.

Reply:
Cool, thanks for the clarification! I was curious, but it's not something I think we need.